bitcoin/src/lib.rs

```diff
@@ -199,7 +199,13 @@ pub mod amount {
 /// Unit parsing utilities.
 pub mod parse {
     /// Re-export everything from the [`units::parse`] module.
-    pub use units::parse::ParseIntError;
+    #[doc(inline)]
+    pub use units::parse::{
+        hex_check_unprefixed, hex_remove_prefix, hex_u128, hex_u128_unchecked, hex_u128_unprefixed,
+        hex_u32, hex_u32_unchecked, hex_u32_unprefixed, int, ContainsPrefixError,
+        MissingPrefixError, ParseIntError, PrefixedHexError, UnprefixedHexError,
+    };
+    // TODO: Add all public structs, enums, and functions
```

Comment:
Thanks for sticking with it, you forgot to remove this comment.

Reply:
that's what i get for working right before bed time lol i'll remove it. Appreciate the patience!